### ISSUE TITLE
How to save the BucketInfo?

### ISSUE THREAD

**Issue by nanquanyuhao** [twonote/radosgw-admin4j#18]
hello, I have a problem, my code like this:
```
Quota bucketQuota = bucketInfo.getBucketQuota();
bucketQuota.setMaxSizeKb(quotaSize);
bucketQuota.setEnabled(true);
bucketInfo.setBucketQuota(bucketQuota);
```
But it does not come into force, what can I do ? Thanks...

**Comment by hrchu** [comment]
Hi @nanquanyuhao, we draft v1.1.0 to support the bucket quota. It is available now on [Github](https://github.com/twonote/radosgw-admin4j/releases). If you prefer to fetch it from maven, plz wait a moment, it needs some time to sync to maven central. Thank you your feedback.

**Comment by nanquanyuhao** [comment]
@hrchu Thanks...

**Comment by hrchu** [comment]
@nanquanyuhao v1.1.0@maven central is ready. Have a try!
https://search.maven.org/#artifactdetails%7Cio.github.twonote%7Cradosgw-admin4j%7C1.1.0%7Cjar

To set quota in bucket scope, plz use [setBucketQuota(String userId, long maxObjects, long maxSizeKB)](https://static.javadoc.io/io.github.twonote/radosgw-admin4j/1.1.0/org/twonote/rgwadmin4j/RgwAdmin.html#setBucketQuota-java.lang.String-long-long-)

Let me know if encounter any problem. Thanks.

**Comment by nanquanyuhao** [comment]
@hrchu Thanks, I have saw the new feature...

But I still have a problem,  like this:
```
PUT /admin/bucket?quota&uid=<uid>&bucket=<bucket-name>&quota
```
How can I `Set Quota for an Individual Bucket`

**Comment by hrchu** [comment]
Oh I missed this since it is described in Ceph master branch only. I am going to add it.

**Comment by hrchu** [comment]
Hi @nanquanyuhao , the "Set Quota for an Individual Bucket" feature seems buggy, even using the [script](https://github.com/ceph/ceph/pull/18324#issuecomment-336846155) provided by the author cannot work on the leatest Ceph version.

I will dig it deeper later...

**Comment by nanquanyuhao** [comment]
@hrchu 
In fact, the original version has been able to meet my needs, I just wanted to make my program more stable.

Thank you!

**Comment by hrchu** [comment]
It seems that the feature has not yet released. https://github.com/ceph/ceph/pull/20885
I will add it when the newer Ceph version available, maybe v12.2.5?

Let me know if any questions, thanks!

**Comment by nanquanyuhao** [comment]
@hrchu 
So that's how it's done...
I thought it had been all the time.
